schedule-replicas-by-topology-labels.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Schedule Replicas by Topology Labels
+aliases: ['/docs/dev/how-to/deploy/geographic-redundancy/location-awareness/','/docs/dev/location-awareness/']
```

Comment:
```suggestion
aliases: ['/docs/dev/location-awareness/','/docs/dev/how-to/deploy/geographic-redundancy/location-awareness/','/tidb/dev/location-awareness']
```

Reply:
Done in https://github.com/pingcap/docs/pull/3825/commits/18a44741a141609bd277bb8d3cdfec57199f8a2d